modules/ingestor/src/graph/sbom/tests.rs

```diff
@@ -45,7 +44,7 @@ async fn query_sboms() -> Result<(), anyhow::Error> {
         )
         .await?;
 
-    let other_sbom = system
+    let _other_sbom = system
```

Comment:
Do we have to assign these? Seems like a smell.

Reply:
also, just carried over, except Clippy started to complain. I dunno why.  Took the easy path to quiet Clippy.